src/internal/kopia/conn.go

```diff
@@ -131,16 +134,21 @@ func (w *conn) Connect(ctx context.Context) error {
 
 func (w *conn) commonConnect(
 	ctx context.Context,
+	opts control.Options,
 	configDir string,
 	bst blob.Storage,
 	password, compressor string,
 ) error {
-	var opts *repo.ConnectOptions
+	kopiaOpts := &repo.ConnectOptions{
+		ClientOptions: repo.ClientOptions{
+			Username: opts.User,
+			Hostname: opts.Host,
```

Comment:
is it okay for these to be zero valued?  Do we need a sane default established in `control.Default()`?  (wrote this before seeing the tests below; seems safe, but still want to sanity check.)

Likewise, is there anything we need to do for common integration testing cases here?

Reply:
fine for them to have a zero value, that's essentially what's happening right now

If we do integration testing I'd much rather have something mock KW and use the mock to ensure the passed in values are what are expected. The tests in this PR already show that KW will behave as expected when given different values and there's no multi-component complexity here that would require a non-mocked integration test (i.e. it's not like details merging where several different components touch the data and we're only really interested in the end result)

As of now, we don't have any infrastructure for mocking KW or other things in corso's kopia package